### PR TITLE
Manual proof mode, WislLifter map insertion resilience, fix tooltips

### LINKED ISSUE #154: Fix ExecMap tooltips
These broke at some point. Fix them :))

### LINKED ISSUE #157: Add a config switch to enable manual proofs when debugging
- [x] Add a command-line switch to `debugverify` to enable manual proof
- [x] Add a config option to the VSCode debugger extension to use that switch
- [x] When debugging `list_iter.wisl` with manual proof, and a manual fold has been added before the `if`, an error occurs - investigate and fix this

### PR DESCRIPTION
**Pull request by NatKarmios**
Resolves #157
Resolves #154

~~Depends on #156 - merge that one first!~~

### CODE REVIEW

**Comment by giltho** [comment]
Can you rebase that please? :) Thank you!

File: GillianCore/debugging/debugger/debugger.ml [hunk 1]
```diff
@@ -716,17 +716,14 @@ struct
...
-      match proc_name with
-      | None -> ()
-      | Some proc_name -> Config.Verification.set_procs_to_verify [ proc_name ]
-    in
+    Fmt_tty.setup_std_outputs ();
+    Config.current_exec_mode := Verification;
+    PC.initialize Verification;
+    Config.stats := false;
+    Config.lemma_proof := true;
+    (match proc_name with
```
Comment (giltho) [inline GillianCore/debugging/debugger/debugger.ml:724]: not a fan of `let () = ` anymore? 😛 
The only reason I use `let ()` is the `if` annoyance
```ocaml
if b then f(); g()
```
and 
```ocaml
if b then let () = f() in g()
```
mean different things and I hate it 🤦

**Review by giltho — APPROVED** [review]

File: GillianCore/debugging/debugger/debugger.ml (hunk 1 above)
Comment (NatKarmios) [reply GillianCore/debugging/debugger/debugger.ml:724]: Hm, in that situation I'd use
```ocaml
if b then (f(); g())
```

Comment (giltho) [reply GillianCore/debugging/debugger/debugger.ml:724]:
```ocaml
if b then begin f(); g() end
```
hahaha

Comment (NatKarmios) [reply GillianCore/debugging/debugger/debugger.ml:724]: Huh, first I've heard of that, that's pretty nice. I'll use that on this situation then hah

**Pull request merged**